### PR TITLE
Feature/sidenav tpmti 67

### PR DESCRIPTION
**Pull request by Dantiff**
[TPMTI-67](https://technekes.atlassian.net/browse/TPMTI-67)

@ben-dalton @bankai254 @hamholla

### CODE REVIEW

File: webpack.dev.js [hunk 1]
```diff
@@ -15,5 +15,9 @@ module.exports = merge(common, {
   plugins: [new webpack.HotModuleReplacementPlugin()],
   optimization: {
     usedExports: true,
+    splitChunks: {
```
Comment (ben-dalton) [inline webpack.dev.js]: What does this do?

Comment (Dantiff) [reply webpack.dev.js]: Oh, we were trying to optimize compile time during dev, unfortunately did not work. I will get rid of this.

**Comment by johnny-five** [comment]
I, @ben-dalton, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket](https://technekes.atlassian.net/browse/tpmti-67)
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Sidenav/SideNavItemToggle.component.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    onClick(e);
+  };
+  return isToggleVisible ? (
+    <SSideNavItemToggle
+      role="side-nav-toggle"
+      {...props}
+      onClick={(e: any) => handleClick(e)}
+    >
+      {children}
+      {!isToggleOpen && (openContent || <div>{'>'}</div>)}
```
Comment (ben-dalton) [inline src/Sidenav/SideNavItemToggle.component.tsx]: Let's go ahead and import icons as the default for these arrows. I think one of the "chevron" icons should work.

File: src/Sidenav/SideNavToggle.component.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    afterToggle ? afterToggle(e) : noop;
+  };
+  return (
+    <SSideNavToggle
+      role="side-nav-toggle"
+      {...props}
+      onClick={(e: any) => handleClick(e)}
+    >
+      {children}
+      {!isOpen && (openContent || <div>{'>'}</div>)}
```
Comment (ben-dalton) [inline src/Sidenav/SideNavToggle.component.tsx]: let's use icons

File: webpack.dev.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ module.exports = merge(common, {
     hot: true,
   },
   output: {
-    filename: '[name].js',
+    filename: '[name].[contenthash].js',
```
Comment (ben-dalton) [inline webpack.dev.js]: Will the user have to import the component from a file with this hash?

File: src/Sidenav/SideNavItemToggle.component.tsx (hunk 2 above)
Comment (Dantiff) [reply src/Sidenav/SideNavItemToggle.component.tsx]: This has been added

File: src/Sidenav/SideNavToggle.component.tsx (hunk 3 above)
Comment (Dantiff) [reply src/Sidenav/SideNavToggle.component.tsx]: This has been added

File: webpack.dev.js (hunk 4 above)
Comment (Dantiff) [reply webpack.dev.js]: This was a mistake, thank you for capturing that. I was to remove any extra configs that I had added to webpack config. This has been removed.

**Pull request merged**